### PR TITLE
Bouquets: read views

### PR DESCRIPTION
**Pull request by abulte**
Adds a list and detail views for bouquets (aka topics).

Also improves some markdown / clamping rendering.

### CODE REVIEW

File: src/store/BouquetStore.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }),
+  actions: {
+    /**
+     * Filter a list of bouquets related to Ecospheres
+     *
+     * @param {Array} bouquets
+     * @returns {Array}
+     */
+    filter (bouquets) {
+      return bouquets.filter(bouquet => bouquet.extras.is_ecospheres || bouquet.tags.includes("ecospheres"))
```
Comment (streino) [inline src/store/BouquetStore.js:18]: We probably don't want to display the main "ecospheres" bouquet, especially when it'll contain thousands of datasets...

File: src/store/BouquetStore.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    /**
+     * Load Ecospheres related bouquets from API
+     *
+     * @returns {Array<object>}
+     */
+    async loadBouquets () {
```
Comment (streino) [inline src/store/BouquetStore.js:25]: So French! But `loadBunches`... We may have to find another term 😅
(not a change request, just me wondering how i18n is going to work when we need it)

File: src/store/BouquetStore.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      if (this.data.length > 0) return this.data
+      let response = await topicsAPI._list()
```
Comment (streino) [inline src/store/BouquetStore.js:27]: We'll need a way to pass a filter to data.gouv otherwise this won't scale.

**Review by streino — APPROVED** [review]

File: src/store/BouquetStore.js (hunk 1 above)
Comment (abulte) [reply src/store/BouquetStore.js:18]: True, I guess we can put the Ecospheres' bouquet id in the config file when it exists, and filter it out. Or add an extra `is_universe` or something.

File: src/store/BouquetStore.js (hunk 2 above)
Comment (abulte) [reply src/store/BouquetStore.js:25]: Yep :p Bouquet is legit though, I checked 🇬🇧

File: src/store/BouquetStore.js (hunk 3 above)
Comment (streino) [reply src/store/BouquetStore.js:27]: Will add it to our shared backlog with data.gouv

**Pull request merged**